cmd/jaeger/config-badger.yaml

```diff
@@ -20,13 +20,15 @@ extensions:
     backends:
       some_store:
         badger:
-          directory_key: "/tmp/jaeger/"
-          directory_value: "/tmp/jaeger/"
+          directories:
+            keys: "/tmp/jaeger/"
+            values: "/tmp/jaeger/"
```

Comment:
I wonder, shouldn't these be different?

Reply:
From the [Badger Documentation](https://dgraph.io/docs/badger/get-started/#opening-a-database)
```
The Dir and ValueDir options are mandatory and must be specified by the client. They can be set to the same value to simplify things.
```